### PR TITLE
Add Kristen Bjorn scraper

### PR DESCRIPTION
**Pull request by Jacintocuenca**

### CODE REVIEW

File: scrapers/KristenBjorn.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      Tags:
+        Name: //a[@role="button"]/text()
+      Performers:
+        Name: $performer//h5/text()
+        URL:
+          selector: $performer/@href
+          postProcess: &appendDomain
+            - replace:
+                - regex: ^(.*)$
+                  with: http://kristenbjorn.com$1
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:35]: simpler
```suggestion
                - regex: ^
                  with: http://kristenbjorn.com
```

File: scrapers/KristenBjorn.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      - kristenbjorn.com/web/model/gay-porn-star/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="scene-item"]/a[@class="scene-box"]
+    scene:
+      Title: //article/div[1]//h2/text()
+      Details: //div[@class="med-text"]/p/text()
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:25]: There are multiple `text()` in some `<p>` due to some `<span>`s
Doing something like this
```yaml
      Details:
        selector: //div[@class="med-text"]/p/text()
        concat: " "
```

or even better 
```suggestion
      Details: //div[@class="med-text"]/p
``` 
is needed to get all of  the description

File: scrapers/KristenBjorn.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          postProcess: &appendDomain
+            - replace:
+                - regex: ^(.*)$
+                  with: http://kristenbjorn.com$1
+      Image: &imageAttr //meta[@property="og:image"]/@content
+      Studio:
+        Name:
+          fixed: Kristen Bjorn
+      Movies:
+        Name: //article/div[5]//h2/text()
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:41]: adding a class to the mix makes it safer (less chance of a false match)
```suggestion
        Name: //article/div[@class="StoreHeaderNewContainer"][2]//h2/text()
```

File: scrapers/KristenBjorn.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      - kristenbjorn.com/gay-porn-star/
+      - kristenbjorn.com/web/model/gay-porn-star/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="scene-item"]/a[@class="scene-box"]
+    scene:
+      Title: //article/div[1]//h2/text()
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:24]:
```suggestion
      Title: //article/div[@class="StoreHeaderNewContainer"][1]//h2/text()
```

adding a class to the mix makes it safer (less chance of a false match)

File: scrapers/KristenBjorn.yml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      Duration:
+        selector: //span[text() = "Running Time:"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: \sh(ou)?rs?\s?
+                with: ':'
+              - regex: \smin(ute)?s?
+                with: ':00'
+              - regex: :$
+                with: :00:00
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:66]: this doesnt relay work,  set log to debug and check when you do some scrapes
i tried the below
```yaml
          - replace:
              - regex: ^(\d+)\s*(?i:h)\D+$ # movies with only hours
                with: $1:00:00
              - regex: ^(\d+)\s*(?i:m)\D+$ # movies with only minutes
                with: '$1:00'
              - regex: ^(\d+)\s*(?i:h)\D+(\d+)\s*(?i:m)\D+$ # movies with hours and minutes
                with: '$1:$2:00'
```
and it seems to work for the 3 different cases listed, if  there are more cases we need to check that again

File: scrapers/KristenBjorn.yml [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+      Synopsis: //div[@class="more"]/text()
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:67]:
```suggestion
      Synopsis: //div[@class="more"]
```
you need the whole div othewise you only get a partial synopsis

File: scrapers/KristenBjorn.yml [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+      Name: //article/div[1]/span/text()
+      Date:
+        selector: //span[text()="Release Date:"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: /19([0-9])
+                with: /$1
+              - regex: /20([0-9])
+                with: /$1
+          - parseDate: 01/02/06
```
Comment (bnkai) [inline scrapers/KristenBjorn.yml:56]: i have a feeling that the dates will be a pain to get correctly
do you have any samples with a 19xx date ?
i found another case where the date is `1/02/06` so the below regex should  be added as well
```yaml
              - regex: ^(\d/\d\d/\d\d)$
                with: 0$1
          - parseDate: 01/02/06
```

File: scrapers/KristenBjorn.yml (hunk 5 above)
Comment (Jacintocuenca) [reply scrapers/KristenBjorn.yml]: Thanks! This seems to work too with the movies I was trying.

File: scrapers/KristenBjorn.yml (hunk 7 above)
Comment (Jacintocuenca) [reply scrapers/KristenBjorn.yml:56]: I couldn't find samples with 19xx. I put it to be safe, but we can always add it back if I find any of such cases, so I'm removing it.

**Pull request merged**